Rakefile

```diff
@@ -86,6 +86,8 @@ end
 
 namespace :reallyenglish do
   require 'yaml'
+  require 'vagrant_cloud'
+  vagrantcloud_user_name = ENV["VAGRANTCLOUD_USERNAME"] || "trombik"
```

Comment:
i do not understand why `VAGRANTCLOUD_USERNAME` is not mandatory and  the `vagrantcloud_user_name` has the default value.

Reply:
https://github.com/reallyenglish/packer-templates/pull/56/commits/2aef74a579811fcfc1481c3d215fdcec11069646 clarifies the reason.